.dockerignore

```diff
@@ -0,0 +1,13 @@
+.dockerignore
+*.pyc
```

Comment:
The files get copied to the docker build environment? They're not just getting recreated?

That seems surprising that the other wildcard patterns would work. Does a single-star not expand to characters, maybe?